### PR TITLE
PIM-10259: Add ArPHP library and fix Arabic RTL in PDF Exports

### PR DESCRIPTION
**Pull request by Balamung**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/Pim/Enrichment/Bundle/PdfGeneration/Builder/DompdfBuilder.php [hunk 1]
```diff
@@ -57,6 +58,16 @@ protected function render($html)
             'chroot' => $this->publicDir
         ]);
         $this->dompdf = new Dompdf($options);
+
+        // See https://github.com/dompdf/dompdf/issues/712#issuecomment-952923539
```
Comment (ghoussard) [inline src/Akeneo/Pim/Enrichment/Bundle/PdfGeneration/Builder/DompdfBuilder.php:62]: Could you put this code block dedicated in method like `formatArabic` or `renderArabic` please ?

**Review by masacc — APPROVED** [review]

**Comment by xavierbriand** [comment]
@ahocquard would you be able to review this PR, please?

File: src/Akeneo/Pim/Enrichment/Bundle/PdfGeneration/Builder/DompdfBuilder.php [hunk 2]
```diff
@@ -70,4 +73,21 @@ protected function output()
...
     }
+
+    /**
+     * Fixes RTL in Arabic texts
+     * see https://github.com/dompdf/dompdf/issues/712#issuecomment-952923539
+     */
+    private function formatArabic(string $html): string
+    {
+        $arabic = new Arabic();
+        $p = $arabic->arIdentify($html);
```
Comment (ahocquard) [inline src/Akeneo/Pim/Enrichment/Bundle/PdfGeneration/Builder/DompdfBuilder.php:84]: what is this variable purpose? Please avoid one letter naming.

File: src/Akeneo/Pim/Enrichment/Bundle/PdfGeneration/Builder/DompdfBuilder.php [hunk 3]
```diff
@@ -70,4 +73,21 @@ protected function output()
     {
         return $this->dompdf->output();
     }
+
+    /**
+     * Fixes RTL in Arabic texts
+     * see https://github.com/dompdf/dompdf/issues/712#issuecomment-952923539
+     */
+    private function formatArabic(string $html): string
```
Comment (ahocquard) [inline src/Akeneo/Pim/Enrichment/Bundle/PdfGeneration/Builder/DompdfBuilder.php:81]: how do you now you fixed the issue? Shouldn't it be a dedicated service to ease the test?

**Review by ahocquard — APPROVED** [review]
GTM after test

When adding a library, a good practice is to indicate in the description that the license is ok (it's ok here).

**Pull request merged**